lms/envs/common.py

```diff
@@ -661,7 +661,7 @@ def _add_microsite_dirs_to_default_template_engine(settings):
 ###############################################################################################
 
 # use the ratelimit backend to prevent brute force attacks
-AUTHENTICATION_BACKENDS = ['ratelimitbackend.backends.RateLimitModelBackend']
+AUTHENTICATION_BACKENDS = ['ratelimitbackend.backends.RateLimitModelBackend', ]
```

Comment:
This isn't really hurting anything, but can this just be reverted so there are no changes for this file?